src/main/java/Erika.java

```diff
@@ -0,0 +1,162 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Erika {
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private static Scanner in = new Scanner(System.in);
+    private static int markIndex;
```

Comment:
The markIndex member of this class can also serve to keep track of the number of tasks in the array, replacing taskArraySize member of classes below. This helps reduce the redundancy in using members.

Reply:
Hi thanks for your comments! I would like to clarify this comment as markIndex is currently used to store the item the user wishes to mark via the "mark"/"unmark" command. I'm not sure how it can be recycled to also store the size of the task array. Thanks for any clarification!